values/nginx-ingress.yaml

```diff
@@ -0,0 +1,101 @@
+# ref: https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/values.yaml
+
+rbac:
+  create: true
+
+livenessProbe:
+  initialDelaySeconds: 30
+  timeoutSeconds: 5
+
+defaultBackend:
+  replicaCount: 1
+
+  resources:
+    limits:
+      cpu: 0.1
+      memory: "20Mi"
+    requests:
+      cpu: 0.1
+      memory: "5Mi"
+
+controller:
+  replicaCount: 1
+
+  publishService:
+    enabled: true
+
+  resources:
+    limits:
+      memory: "512Mi"
+    requests:
+      memory: "128Mi"
+
+  service:
+    annotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "10254"
+
+  config:
+    use-proxy-protocol: "false"
+    enable-vts-status: "true"
+    ssl-redirect: "true"
+    http-snippet: |-
+      upstream engine_dynamic_backend {
+        server 0.0.0.1;
+        balancer_by_lua_block {
+          local balancer = require "ngx.balancer";
+          local ok, err = balancer.set_current_peer(ngx.var.qix_session_host, ngx.var.qix_session_port);
+          if not ok then
+            return ngx.exit(500)
+          end;
+        }
+      }
+    server-snippet: |
+      # global timeouts
+      keepalive_timeout 3h;
+      proxy_read_timeout 1d;
+      proxy_send_timeout 1d;
+
+      location /create_qix_session {
+        internal;
+        proxy_pass http://qix-session.default.svc.cluster.local:9455/v1/session;
+      }
+
+      location /app {
+          set $qix_session_host '';
+          set $qix_session_port '';
+
+          access_by_lua_block {
+            local uri = string.sub(ngx.var.request_uri, 5)
+            local routeResponse = ngx.location.capture("/create_qix_session" .. uri)
+            if routeResponse.status == 200 then
+              local cjson = require "cjson";
+              local qix_session_route_data = cjson.decode(routeResponse.body);
+              ngx.var.qix_session_host = qix_session_route_data.ip;
+              ngx.var.qix_session_port = qix_session_route_data.port;
+              ngx.req.set_header("X-Qlik-Session", qix_session_route_data.sessionId)
+              ngx.req.set_uri("/app/")
+              ngx.log(ngx.INFO, "Session placement: Session placed on " .. qix_session_route_data.ip .. " with id " .. qix_session_route_data.sessionId)
```

Comment:
Do we get this log messages? I think that ngx.NOTICE is the "least" loggable level by default

Reply:
Can't find any info logs, only warn. Will change them to notice.